wsclient/src/main/java/org/hpccsystems/ws/client/platform/ECLExceptionInfo.java

```diff
@@ -1,6 +1,7 @@
 package org.hpccsystems.ws.client.platform;
 
 import org.hpccsystems.ws.client.gen.wsworkunits.v1_73.ECLException;
+import org.hpccsystems.ws.client.gen.wsworkunits.v1_73.EspException;

```

Comment:
Let's make sure we're not mixing up different objects, the ESPException should only occur upon Web Service issues, where as the ECL Exception should be associated w/ WU issues..

If we're willfully combining the 2 types of exceptions, let's be explicit about it, the name of the class should reflect the new object we're representing w/ this class (to be honest, I don't know what this new class is representing, perhaps hpccexceptions?)

Also, this seems like another candidate for the wrappers package

Reply:
@drealeed I'm still hung up on the name of this class.... this combination of exceptions is really related to ECL workunits let's rename to ECLWUExceptions?